### DISCUSSION TITLE
Using Katex in lexical

### DISCUSSION THREAD

**Discussion by Sapkotaanish** [facebook/lexical#2597] [Q&A]
I am trying to integrate Katex in lexical (inside react) but I didn't see it working anywhere except the playground.
Will there be "Katex" integrated version of lexical in future release or should I integrate Katex myself (and any suggestion on how should I approach that)?
Thank you.

**Comment by abstractalgo** [comment]
I've extracted it from the playground example and it worked well - `EquationNode` and an extra bit from `MarkdownShortcutPlugin` to have it triggered by regex `$ ... $`.
